package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@prezly/content-renderer-react-js",
-    "version": "0.28.3",
+    "version": "0.29.0-0",
```

Comment:
Is the Storybook task failure because of this PR or is it just a random fail? 😄

Reply:
It's a random false-positive from Loki. I logged a ticket to consider replacing it with Chromatic, which is a bit more reliable in that regard

Reply:
Alrighty 👍